eclair-core/src/main/scala/fr/acinq/eclair/router/Monitoring.scala

```diff
@@ -17,16 +17,40 @@
 package fr.acinq.eclair.router
 
 import fr.acinq.eclair.router.Router.GossipDecision
+import fr.acinq.eclair.wire.ChannelUpdate
 import fr.acinq.eclair.{LongToBtcAmount, MilliSatoshi, getSimpleClassName}
 import kamon.Kamon
-import kamon.metric.Counter
+import kamon.metric.{Counter, MeasurementUnit}
 
 object Monitoring {
 
   object Metrics {
     val FindRouteDuration = Kamon.timer("router.find-route.duration", "Path-finding duration")
     val RouteLength = Kamon.histogram("router.find-route.length", "Path-finding result length")
 
+    val QueryChannelRangeBlocksCount = Kamon.histogram("router.gossip.query-channel-range.blocks-count", "Number of blocks requested in query-channel-range")
```

Comment:
What is the rationale behind the trailing `-count`? Why does e.g. `NewChannelAnnouncements` not have it?

Reply:
True, I can remove those, they're implicit